src/node_modules/app/components/AccessibilityMenu.svelte

```diff
@@ -24,16 +29,30 @@
 	export let screen;
 
 	let label;
-	let showLabels;
+	let useRadios;
+	let menuWidth;
```

Comment:
minor, for this kind of vars I usually add a comment like `// bound` at the right of the var declaration